app/src/main/java/org/fossasia/openevent/Adapters/TracksListAdapter.java

```diff
@@ -42,15 +53,31 @@ public int getItemCount() {
         return tracks.size();
     }
 
+    private void setAnimation(View viewToAnimate, int position) {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
Ok I got the gist. But for the first load of let's say 20 items out of which 10 are displayed on screen the animation will be jagged because 

* Animation is being decoded from resource xml every time on the UI thread which is unpredictable and can make it slow
* The animation sequence will be 
```
                      item1#startAnimation() ...  ... item1#endAnimation()
                                                   item2#startAnimation()
                                                             item3#startAnimation()
```
Since these animations are not part of any AnimationSet and each animation affects the overall layout, there will be a lot of draw calls which will make the animation stutter, you can check out https://www.youtube.com/watch?v=YCHNAi9kJI4 on why that is a problem.

My advice: remove the animations, we'll add them if we need later on. No need to bother.